masquerade.go

```diff
@@ -79,21 +79,33 @@ type Provider struct {
 	// Url used to vet masquerades for this provider
 	TestURL     string
 	Masquerades []*Masquerade
+
+	// SNIConfig is a map of SNIs with configurations per region.
+	// The key can be a region initial or the default value, used when
+	// the region wants to use arbitrary SNIs but doesn't provide the SNI list.
```

Comment:
I don't see any use of region qualifiers here.  That's fine - I think that can be done in flashlight* - but maybe this comment is outdated?

*maybe you've already done that; I haven't looked at the flashlight PR yet

Reply:
Aaaaaah yes, outdated comment, I've moved to flashlight the responsibility to retrieve the region and pass to fronted the SNIConfig. I'll rewrite this comment